envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/BUILD

```diff
@@ -4,8 +4,7 @@ wasm_cc_binary(
     name = "mgw-websocket.wasm",
     srcs = ["filter.cc", "filter.h", "handler_impl.h", "handler_impl.cc", "handler.h"],
     deps = [
-        "@proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics",
         "@mgw_wasm_websocket_api//:api_cc_proto",
-        "@proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics_lite"
+        "@proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics_full"
```

Comment:
@VirajSalaka pls confirm on this dependency update

Reply:
Can't exactly recall why it was added previously. As long as it builds and does the job, we are good. This does not affect regular REST calls.